packages/cli/src/commands/index.ts

```diff
@@ -2,7 +2,71 @@ export * from "./build";
 export * from "./codegen";
 export * from "./create";
 export * from "./deploy";
-export * from "./infra";
-export * from "./run";
 export * from "./docgen";
+export * from "./infra";
 export * from "./manifest";
+export * from "./run";
+export * from "./types";
+
+import { BuildCommandOptions } from "./build";
+import { CodegenCommandOptions } from "./codegen";
+import {
+  CreateCommandOptions,
+  SupportedAppLangs,
+  SupportedPluginLangs,
+  SupportedWasmLangs
+} from "./create";
+import { DeployCommandOptions } from "./deploy";
+import {
+  DocgenCommandOptions,
+  DocgenActions
+} from "./docgen";
+import {
+  InfraCommandOptions,
+  InfraActions
+} from "./infra";
+import {
+  ManifestSchemaCommandOptions,
+  ManifestMigrateCommandOptions,
+  ManifestType
+} from "./manifest";
+import { RunCommandOptions } from "./run";
+
+export interface CommandTypings {
```

Comment:
I’d place this into its own `CommandTypings.ts` file and use `index.ts` as a barrel, instead of defining it in `index.ts`.

Reply:
So the idea with having it here is that it will "force" developers to keep it updated as new commands are added. If we move this into a separate file I'm worried it will be "out of sight and out of mind" and it won't be updated.

If it doesn't get updated, then the typing-magic that ensures the cli-js package is correct will not work, and the cli-js package will fall out of sync w/ the CLI's actual interface.